routes/api.php

```diff
@@ -16,3 +16,5 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::post('/reading-items/new', 'ReadingItemController@store');
```

Comment:
doesn't require auth ?

Reply:
is this for adding new reading-items ? shouldn't we protect this route ?

Reply:
Created a tracking for this #3

Reply:
Yea I'm not sure on what kind of authentication, however my gut says I wanna stick with the standard OAuth2.0.